### PR TITLE
Backported functions for enabling reporting dumps (3.12)

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4565/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4565/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4565/

File: libutils/sequence.c [hunk 1]
```diff
@@ -463,6 +464,8 @@ Seq *SeqFromArgv(int argc, const char *const *const argv)
     return args;
 }
 
+
+
```
Comment (vpodzime) [inline libutils/sequence.c]: No blank lines needed here, AFAICT.

Comment (olehermanse) [reply libutils/sequence.c]: Fixed, thanks.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4567/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4567/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4567/

**Pull request merged**